RiotNSE/NotificationService.swift

```diff
@@ -583,16 +583,13 @@ class NotificationService: UNNotificationServiceExtension {
                             break
                     }
                     
-<<<<<<< HEAD
                     guard notificationBody != nil else {
                         MXLog.debug("[NotificationService] notificationContentForEvent: notificationBody is nil")
                         onComplete(nil, false)
                         return
                     }
```

Comment:
@giomfo do you know if we need to keep this change for Tchap ?

Reply:
Done in following commit.